examples/cosmos/ics20/README.md

```diff
@@ -39,5 +183,12 @@ if source {
 }
 ```
 
+### ICS 20
```

Comment:
Is this header meant to have this name? I found it confusing.

Reply:
Could be confusing, I agree. I was trying to group by file, so I've just replaced it with the literal file name.